src/benchmarks/micro/corefx/System.Net.NetworkInformation/NetworkInterfaceTests.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Reflection;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Net.NetworkInformation.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class NetworkInterfaceTests
+    {
+        private PropertyInfo[] NetworkInterfaceInfo = typeof(System.Net.NetworkInformation.NetworkInterface).GetProperties(BindingFlags.Instance | BindingFlags.Public);
+        private PropertyInfo[] IPInterfaceStatisticsInfo = typeof(System.Net.NetworkInformation.IPInterfaceStatistics).GetProperties(BindingFlags.Instance | BindingFlags.Public);
+        private PropertyInfo[] IPInterfacePropertiesInfo = typeof(System.Net.NetworkInformation.IPInterfaceProperties).GetProperties(BindingFlags.Instance | BindingFlags.Public);
+
+        [Benchmark]
+        public void GetAllNetworkInterfaces() => NetworkInformation.NetworkInterface.GetAllNetworkInterfaces();
```

Comment:
You should return the benchmark result to prevent from possible [dead code elimination](https://github.com/dotnet/performance/blob/master/docs/microbenchmark-design-guidelines.md#dead-code-elimination)

```suggestion
        public NetworkInterface[] GetAllNetworkInterfaces() => NetworkInterface.GetAllNetworkInterfaces();
```

Reply:
done.